 
int64 argcRFID
string addressRFID
---

bool status

 
